pkg/collector/listeners/docker.go

```diff
@@ -329,21 +307,57 @@ func (s *DockerService) GetADIdentifiers() ([]string, error) {
 }
 
 // GetHosts returns the container's hosts
-// TODO
 func (s *DockerService) GetHosts() (map[string]string, error) {
-	return s.Hosts, nil
+	if s.Hosts != nil {
+		return s.Hosts, nil
+	}
+
+	ips := make(map[string]string)
+	c := docker.Container{ID: string(s.ID)}
+
+	cInspect, err := c.Inspect(false)
```

Comment:
Probably better to directly use docker.Inspect(string(s.ID), false) in this case